components/tags/index.css

```diff
@@ -74,3 +76,14 @@ governing permissions and limitations under the License.
   white-space: nowrap;
   text-overflow: ellipsis;
 }
+
+.spectrum-Tags-item--removable {
+
+  .spectrum-Tags-itemClearButton {
+    &.is-focused {
+      border: 2px solid;
```

Comment:
no DNA var for that border? like this one maybe ? http://spectrograph.corp.adobe.com/token-browser/token/36222?q=tag%20removeable